modules/svg/include/SkSVGApi.h

```diff
@@ -0,0 +1,15 @@
+#ifndef SkSVGAPI_DEFINED
+#define SkSVGAPI_DEFINED
+
+#include <string>
+#include <vector>
+
+class __attribute__((visibility("default"))) SkSVGApi {
+ public:
+    static bool registerFonts(const std::vector<std::string>& fontFilePaths);
+    static bool registerFonts(const std::vector<std::pair<std::string, std::string>>& fontFilePaths);
+
+    static bool renderSvg(const std::string& svg, const std::string& outputFilePath);
```

Comment:
Simplest way is to let skia encode and write to disk, even though this isn't a true performance measurement